ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/packages/index.tsx

```diff
@@ -18,28 +18,145 @@
  */
 
 import { createFileRoute } from '@tanstack/react-router';
+import {
+  createColumnHelper,
+  getCoreRowModel,
+  useReactTable,
+} from '@tanstack/react-table';
 
+import { usePackagesServiceGetPackagesByRunId } from '@/api/queries';
+import { prefetchUseRepositoriesServiceGetOrtRunByIndex } from '@/api/queries/prefetch';
+import { useRepositoriesServiceGetOrtRunByIndexSuspense } from '@/api/queries/suspense';
+import { Package } from '@/api/requests';
+import { DataTable } from '@/components/data-table/data-table';
 import { LoadingIndicator } from '@/components/loading-indicator';
+import { ToastError } from '@/components/toast-error';
 import {
   Card,
   CardContent,
   CardDescription,
   CardHeader,
   CardTitle,
 } from '@/components/ui/card';
-import { Label } from '@/components/ui/label';
+import { toast } from '@/lib/toast';
+import { paginationSchema } from '@/schemas';
+
+const defaultPageSize = 10;
+
+const columnHelper = createColumnHelper<Package>();
+
+const columns = [
+  columnHelper.accessor(
+    (pkg) => {
+      const { type, namespace, name, version } = pkg.identifier;
+      return `${type ? type.concat(':') : ''}${namespace ? namespace.concat('/') : ''}${name ? name : ''}${version ? '@'.concat(version) : ''}`;
+    },
+    {
+      id: 'package',
+      header: 'Package ID',
+      cell: ({ row }) => {
+        return <div className='font-semibold'>{row.getValue('package')}</div>;
+      },
+    }
+  ),
+  columnHelper.accessor('processedDeclaredLicense.spdxExpression', {
+    id: 'declaredLicense',
+    header: 'Declared License',
+    cell: ({ row }) => (
+      <div className='break-all'>{row.getValue('declaredLicense')}</div>
+    ),
+  }),
```

Comment:
Some earlier nested object accessor key produced some console errors if the item was not found, didn't it? Does this give them as well, so should it be accessorFn instead?